packages/contracts-rfq/contracts/FastBridgeV2.sol

```diff
@@ -123,6 +150,7 @@ contract FastBridgeV2 is Admin, IFastBridgeV2, IFastBridgeV2Errors {
             params.destAmount,
             params.sendChainGas
         );
+        emit BridgeRequestedV2(transactionId, paramsV2.quoteId);
```

Comment:
Do you mean the second event naming? It does make sense to me to rename this honestly, since `BridgeRequsted` was originally meant for the relayers needs more so than for the indexers.

Reply:
Renamed in e1301271526b854c456727b123ef2293b684a260